.github/workflows/release.yml

```diff
@@ -203,6 +203,7 @@ jobs:
                 m4 \
                 perl \
                 pkg-config \
+                python-is-python3 \
```

Comment:
make sure this is up-to-date with DEVELOPER.MD

Reply:
I don't think this is actually required. Let me test and see.
I originally added it as a troubleshooting step to account for this error/warning: https://github.com/Kong/kong/actions/runs/10726727011/job/29747428984#step:17:341

Reply:
Excluding this didn't seem to make a difference for the arm64 build as seen here: https://github.com/Kong/kong/actions/runs/10745604128/job/29804939887
So I've pushed a new commit to remove it.